core/src/test/java/org/apache/accumulo/core/file/rfile/RFileTest.java

```diff
@@ -1787,137 +1789,146 @@ private void runVersionTest(int version, AccumuloConfiguration aconf) throws IOE
     reader.close();
   }
 
-  public static AccumuloConfiguration getAccumuloConfig(String cryptoConfSetting) {
-    return new SiteConfiguration(CryptoTest.class.getClassLoader().getResource(cryptoConfSetting));
+  public static AccumuloConfiguration getAccumuloConfig(boolean cryptoOn) {
+    ConfigurationCopy cfg = new ConfigurationCopy(DefaultConfiguration.getInstance());
+    if (cryptoOn) {
```

Comment:
Bools are frustrating, because they don't carry semantic intent. Java 8 supports switch statements on String now, so you could just use a switch statement here, and that makes the patch significantly smaller.

Or you could use an enum instead of Strings. Either way, the patch becomes much smaller if you can keep the `CryptoTest.CRYPTO_ON_CONF` and `CryptoTest.CRYPTO_OFF_CONF` constants throughout.

Reply:
I like the simplicity of boolean but I guess I am spoiled by Intellij... When I look at a function I see: getAccumuloConfig(cryptoOn: true);

In this case, keeping it a string would at least minimize the changes.